conf/Config.js

```diff
@@ -48,15 +48,18 @@ class Config {
             });
         }
 
-        // config is validated, safe to assign directly to the config object
-        Object.assign(this, parsedConfig);
-
         // whitelist IP, CIDR for health checks
-        this.healthChecks = { allowFrom: ['127.0.0.1/8', '::1'] };
-        const allowedHealthChecks = parsedConfig.server.healthChecks.allowFrom;
-        if (allowedHealthChecks) {
-            this.healthChecks.allowFrom.concat(allowedHealthChecks);
+        const defaultHealthChecks = ['127.0.0.1/8', '::1'];
+        const healthChecks = parsedConfig.server.healthChecks;
+        if (healthChecks.allowFrom.length > 0) {
```

Comment:
You should not need this special case check, since `concat` on an empty array should work as well (though not if `allowFrom` is not an array, of course)